### DISCUSSION TITLE
Support Sibionics new CGM

### DISCUSSION THREAD

**Discussion by Akenjeru** [j-kaltes/Juggluco#222] [General]
sibionics has a new sensor, it works similarly to dexcom, there is a separate sensor and a transmitter installed separately. Is support possible in the future? or do I need to send it to you to understand?
![photo_2024-11-12_01-57-21](https://github.com/user-attachments/assets/a7f3ac56-880e-433e-99fe-ca5a24fd40c9)

**Comment by UberQueiroz** [comment]
Those "detachable" sensor are not higienic and also can be ripped apart more easily. Too bad Sibionics is going that way.

**Comment by Akenjeru** [comment]
I have this sensor, can you provide any data so that I can add this sensor to Juggluco? It doesn't even have to be 24 days, let it work its 14 days, the main thing is that there is support for Juggluco and there it can be transferred to Xdrip
